test/extended/util/certs/info/generate.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/origin/pkg/certs"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/image"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Fprintf(os.Stderr, "error: requires exactly one argument\n")
+		os.Exit(1)
+	}
+	outputDir := os.Args[1]
+
+	image.InitializeImages("")
```

Comment:
remove?

Reply:
Its necessary for `exutil.FixturePath` outside of the test. I'll rework it to avoid bindata